### PR TITLE
Zerosizetoggle

### PR DESCRIPTION
**Pull request by turtletooth**
Zero thickness lines are really useful for a variety of purposes, but there was no quick way to toggle back and forth between seeing them and hiding them.  I made a toggle and context menu item for them.

https://youtu.be/rUGjIkgdvAU

### CODE REVIEW

**Comment by konero** [comment]
Thank you. :+1: 

If you have the time I issued a similar proposal #433.

**Comment by blurymind** [comment]
This is super useful!! Please merge this :D

**Comment by shun-iwasawa** [comment]
For now this branch conflicts to master branch since we applied clang-format on v1.0.3 release.
Sorry to trouble you but could you please apply clang-format (with [toonz/sources/.clang-format](https://github.com/opentoonz/opentoonz/blob/master/toonz/sources/.clang-format)) to your PR or rebase your commits to the current master?
Thank you for your contributions.

**Comment by turtletooth** [comment]
Updated to 1.0.3

**Comment by shun-iwasawa** [comment]
jenkins

File: toonz/sources/toonz/frameheadgadget.h [hunk 1]
```diff
@@ -86,4 +87,18 @@ class FilmstripFrameHeadGadget : public FrameHeadGadget {
   bool eventFilter(QObject *obj, QEvent *event);
 };
 
+namespace ZeroThickToggleGui {
```
Comment (shun-iwasawa) [inline toonz/sources/toonz/frameheadgadget.h]: @turtletooth 
I think the implementation is fine. But IMO these code should be moved from frameheadgadget.h/.cpp to sceneviewercontextmenu.h/.cpp as this feature will be used only in the viewer's context menu. The other part of frameheadgadget sources are for the Level Strip, so mixing them may cause confusion.

**Comment by turtletooth** [comment]
@shun-iwasawa That makes sense.  I made the changes requested.

**Comment by shun-iwasawa** [comment]
Jenkins

**Comment by shun-iwasawa** [comment]
Jenkins, again

**Comment by shun-iwasawa** [comment]
LGTM

**Pull request merged**